net/haproxy/src/opnsense/mvc/app/models/OPNsense/HAProxy/HAProxy.xml

```diff
@@ -567,7 +567,7 @@
                     <MinimumValue>1</MinimumValue>
                     <MaximumValue>65535</MaximumValue>
                     <ValidationMessage>Please specify a value between 1 and 65535.</ValidationMessage>
-                    <Required>Y</Required>
+                    <Required>N</Required>
```

Comment:
it needs a double-check in the template that an empty value is not harmful to the generated config and the default of 80 probably also needs to be removed above.